source/hello.js

```diff
@@ -10,10 +14,14 @@ export default React => stampit(React, {
   propTypes: {
     actions: PropTypes.shape({
       setWord: func.isRequired,
-      setMode: func.isRequired,
```

Comment:
@ericelliott Why no comma dangle? They make diffs more sane and require less line changes when expanding.